src/util/piped_process.cpp

```diff
@@ -168,7 +168,8 @@ piped_processt::piped_processt(const std::vector<std::string> &commandvec)
   {
     throw system_exceptiont("Input pipe creation failed for child_std_IN_Wr");
   }
-  if(!SetHandleInformation(child_std_IN_Rd, HANDLE_FLAG_INHERIT, 0))
+  if(!SetHandleInformation(
```

Comment:
I don't understand this change (I've read the commit message). Care to elaborate on what's changed here?